packages/server/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -ex
+
+dir=$(pwd)
+
+echo "prout $dir "
+echo "$dir/packages/server/dist/server.js"
+
+# https://docs.docker.com/engine/containers/multi-service_container/
+
+# connect ui
+if [ "$FLAG_SERVE_CONNECT_UI" == "true" ]; then
+  node "$dir/packages/server/dist/server.js" &
+
+  # This is not recommended, you should server UI from a dedicated static website hosting
+  npm run -w @nangohq/connect-ui prod:unsafe &
+
+  # Wait for any process to exit
+  wait -n
+
+  # Exit with status of process that exited first
+  exit $?
```

Comment:
not sure the background processes are being killed when the script exits. We might want to trap signals and ensure proper cleanup

Reply:
that's the recommended way to run multi process in container. I don't know enough about bash to know if it's good enough
https://docs.docker.com/engine/containers/multi-service_container/

Reply:
it is indeed running in a container so it is maybe an overkill. A less naive approach could be something like this:
```
...
# trap signals and kills processes
trap 'kill $server_pid $ui_pid; exit' SIGINT SIGTERM

# wait for processes to exit
wait -n
exit_code=$?

# clean up processes
kill $server_pid $ui_pid 2>/dev/null || true
exit $exit_code
```

Reply:
not sure how it behaves with multi-process, I'll merge as-is and if get complain I'll investigate